target/linux/ar71xx/config-4.4

```diff
@@ -47,6 +47,7 @@ CONFIG_ATH79_MACH_AP136=y
 CONFIG_ATH79_MACH_AP143=y
 CONFIG_ATH79_MACH_AP147=y
 CONFIG_ATH79_MACH_AP152=y
+CONFIG_ATH79_MACH_AP531B0=y
```

Comment:
You have changed file attributes (`100644 → 100755`) which is **wrong** - please, fix it.